data/data.json

```diff
@@ -169,7 +169,7 @@
                "ShortDescriptionLine2":"See http://www.radio106.co.il",
                "Poster":"KolYizrael106.png",
                "StreamUrls":[
-		   "http:/yezreel.cdnwz.net/yezreel"
+                  "http:/yezreel.cdnwz.net/yezreel"
```

Comment:
I don't know why we have it. I did not touch this part of the code and I used JSON formatter to do the formatting